lark/parsers/earley.py

```diff
@@ -281,7 +282,7 @@ def parse(self, lexer, start):
         # If the parse was successful, the start
         # symbol should have been completed in the last step of the Earley cycle, and will be in
         # this column. Find the item for the start_symbol, which is the root of the SPPF tree.
-        solutions = [n.node for n in columns[-1] if n.is_complete and n.node is not None and n.s == start_symbol and n.start == 0]
+        solutions = list(OrderedSet([n.node for n in columns[-1] if n.is_complete and n.node is not None and n.s == start_symbol and n.start == 0]))
```

Comment:
We have `lark.utils.dedup_list` which does this exact operation, but it should be slightly faster (and more explicit).

Reply:
Good catch, thanks!